swan-lake/learn-the-platform/run-in-the-cloud/function-as-a-service/azure-functions.md

```diff
@@ -138,30 +124,35 @@ This section uses different types of triggers and bindings to build Azure functi
 
 ### HTTP Trigger -> queue output
 
-The following Ballerina code gives an example of using an HTTP trigger to invoke the function, a queue output binding to write an entry to a queue, and also an HTTP output binding to respond to the caller with a message. 
+The following Ballerina code gives an example of using an HTTP trigger to invoke the function, a queue output binding to write an entry to a queue.
 
 First, create a queue to hold the outputs of the function by accessing the storage account that was created alongside the function app in the prerequisites. Select **Queues** in the sidebar in the storage accounts. Click the **Add queue** button, and enter the same value as the value of the `queueName` property in the below `QueueOutput` annotation.
 
 ```ballerina
+import ballerina/http;
 import ballerinax/azure_functions as af;
 
-@af:Function
-public function fromHttpToQueue(af:Context ctx, 
-                                @af:HTTPTrigger { authLevel: "anonymous" } af:HTTPRequest req, 
-                                @af:QueueOutput { queueName: "queue1" } af:StringOutputBinding msg) 
-                                returns @af:HTTPOutput af:HTTPBinding {
-    msg.value = req.body;
-    return { statusCode: 200, payload: "Request: " + req.toString() };
+public type Person record {
+    string name;
+    int age;
+};
+
+service / on new af:HttpListener() {
+    resource function post queue(@http:Payload Person person) returns @af:QueueOutput {queueName: "people"} string {
+        return person.name + " is " + person.age.toString() + " years old.";
+    }
 }
 ```
 
-Build the package by executing the `bal build` command on the package directory, and then, deploy it using the `az cli` command shown in the Ballerina build output as in the previous section.
+Build the package by executing the `bal build` command on the package directory, and then, deploy it using the `func` command shown in the Ballerina build output as in the previous section.
```

Comment:
```suggestion
Execute the `bal build` command on the package directory to build the package. Then, execute the `func` command shown in the Ballerina build output as in the previous section to deploy it.
```
Better if we can mention the full command here rather than asking to refer the previous section.